### ISSUE TITLE
Accessibility Testing for Health Enrollment: 1010 team, 1010EZ, Household section

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#58694]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

David Kennedy

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

This testing went well. We're relying on the design system and I didn't see anything that veered away from those standards.

### Who completed the axe scans?

David Kennedy

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

I found no errors when testing with Axe.

### How did axe testing go?

Somewhat related, when looking at the HTML structure, I noticed we use a lot of nested fieldsets in this new section of the 1010EZ and elsewhere on the form. While this isn't a WCAG violation, I've [filed a ticket for an accessibility enhancement](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57406) since nested fieldsets can cause issues for some people who use screen readers.

### Who completed the content zoom and reflow test?

David Kennedy

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

This testing went well. I did notice some lack of padding on the lefthand side at 200 percent, but nothing that would cause readability issues.

### Who completed the keyboard navigation test?

David Kennedy

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [ ] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Overall, the testing went well. I did find an issue with our modal for canceling adding dependents. It's [noted in this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/58522). When the modal closes, it doesn't return focus back to the right place. That's the only issue I found.

Another thing I've noted, and plan to talk to our engineers about: When you get to the add a dependent section and advance in steps, focus moves to the the container element of the fieldset. Most of the time, this works fine as those are labeled with a legend. However, it's inconsistent with the rest of the form, which shifts focus to the h2 on the page.

### Do you have any other results to share?

I also completed screen reader testing with these pairings:

- Safari/VoiceOver on MacOS
- Firefix/NVDA on Windows
- Chrome/JAWS on Windows

I didn't find any major issues through this testing. Also, I have [this testing logged in TestRail](https://dsvavsp.testrail.io/index.php?/runs/view/4610&group_by=cases:section_id&group_order=asc&group_id=8584).
